admin_app/src/components/ContentCard.tsx

```diff
@@ -29,21 +30,25 @@ export const ContentCard: React.FC<ContentCardProps> = (
     >
       {/* Icons Container */}
       <div className="absolute flex top-2 right-2 ">
-        <PencilIcon
-          data-modal-target="edit-content-modal"
-          data-modal-toggle="edit-content-modal"
-          className="w-3 h-3 text-gray-500 cursor-pointer hover:text-red-200"
-          onClick={() =>
-            card.editMe({
-              content_id: card.content_id,
-              content_text: card.content_text,
-            })
-          }
-        />
-        <TrashIcon
-          className="w-3 h-3 text-gray-500 cursor-pointer hover:text-red-200"
-          onClick={() => card.deleteMe(card.content_id)}
-        />
+        {card.showEditButton ? (
+          <>
+            <PencilIcon
```

Comment:
Opinion: The Edit and Delete buttons could be twice as big. Right now, it is a bit difficult to click on them.

Reply:
Added to [AAQ-119](https://idinsight.atlassian.net/browse/AAQ-119)